java/src/test/java/vowpalWabbit/learner/VWLearnersTest.java

```diff
@@ -67,6 +68,15 @@ public void testBadModel() {
     }
 
     @Test
+    public void testSaveModel() {
+        VWLearner vw = VWLearners.create("--quiet");
+        File file = new File("/tmp/saved_test_model");
```

Comment:
I didn't know about TemporaryFolder, it definitely is convenient. I've made the change.